File: sdk/ios/Sources/DuallerSDK/CanvasHandler.swift
import UIKit

/// Handles canvas operations (delegated to WebView)
public final class CanvasHandler: Handler {
    public let apiName: String = "canvas"

    public func handle(_ message: BridgeMessage) -> String? {
        // Canvas operations are delegated to the WebView
        return "{}"
    }
}

File: sdk/android/src/main/java/com/dualler/sdk/handler/CanvasHandler.kt
package com.dualler.sdk.handler

import com.dualler.sdk.protocol.BridgeMessage

/**
 * Handles canvas drawing operations.
 * Delegates to Android Canvas API.
 */
class CanvasHandler : Handler {
    override val apiName: String = "canvas"

    override fun handle(message: BridgeMessage): String? {
        // Canvas operations are typically executed on the UI thread
        // via the WebView. This handler records commands.
        return "{}"
    }
}
